wrappers/modules/output.nix

```diff
@@ -43,6 +43,14 @@ in {
       '';
     };
 
+    defaultEditor = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Configure <command>nvim</command> as the default editor using the EDITOR environment variable
```

Comment:
Shouldn't we simply use `nvim` here (because the doc is generated from markdown) ?